sound/pci/hda/patch_realtek.c

```diff
@@ -2044,6 +2046,24 @@ static void alc1220_fixup_clevo_p950(struct hda_codec *codec,
 	snd_hda_override_conn_list(codec, 0x1b, 1, conn1);
 }
 
+/* fix the default Auto-Mute Mode in mixer */
```

Comment:
please be more descriptive in this comment. What is the default auto-mute mode set to, and what is wrong with that configuration?
Which mode do we switch to instead?

Reply:
I have updated the alc887_fixup_asus_automute_mode function's comment and the commit message. Hope these will make it more clearly.